astropy/io/fits/tests/test_fitsheader.py

```diff
@@ -121,13 +121,33 @@ def test_fitsort(self, capsys):
         assert out[3].endswith('test0.fits 19/05/94')
 
         # check that COMMENT and HISTORY are excluded
-        fitsheader.main(['-e', '0', '-f', self.data('tb.fits')])
+        fitsheader.main(['-f', '-e', '0', self.data('tb.fits')])
```

Comment:
Why this change?

Reply:
It'll interpret the filename as the argument to `-f` otherwise. I'm not sure if there's a way around this with argparse, apart from a separate `sort_key` argument...

Reply:
Does sound like a breaking change. Let's see what subpackage maintainer(s) think.